docs/whats-new/2021-02.md

```diff
@@ -0,0 +1,161 @@
+- [Introduction to porting apps to .NET Core](../architecture/porting-existing-aspnet-apps/introduction.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Logging differences between ASP.NET MVC and ASP.NET Core](../architecture/porting-existing-aspnet-apps/logging-differences.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Compare middleware to modules and handlers](../architecture/porting-existing-aspnet-apps/middleware-modules-handlers.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Migrate to ASP.NET Core 2.1](../architecture/porting-existing-aspnet-apps/migrate-aspnet-core-2-1.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Migrate large solutions to ASP.NET Core](../architecture/porting-existing-aspnet-apps/migrate-large-solutions.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Strategies for migrating ASP.NET Web Forms apps](../architecture/porting-existing-aspnet-apps/migrate-web-forms.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Migration considerations](../architecture/porting-existing-aspnet-apps/migration-considerations.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Compare Razor usage in ASP.NET MVC and ASP.NET Core](../architecture/porting-existing-aspnet-apps/razor-differences.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Routing differences between ASP.NET MVC and ASP.NET Core](../architecture/porting-existing-aspnet-apps/routing-differences.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Serve static files in ASP.NET MVC and ASP.NET Core](../architecture/porting-existing-aspnet-apps/serving-static-files.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Compare ASP.NET SignalR and ASP.NET Core SignalR](../architecture/porting-existing-aspnet-apps/signalr-differences.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Strategies for migrating while running in production](../architecture/porting-existing-aspnet-apps/strategies-migrating-in-production.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Summary](../architecture/porting-existing-aspnet-apps/summary.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Compare testing options between ASP.NET MVC and ASP.NET Core](../architecture/porting-existing-aspnet-apps/testing-differences.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Understand and update dependencies](../architecture/porting-existing-aspnet-apps/understand-update-dependencies.md) - Porting ASP.NET eBook REVIEW DRAFT
+- [Compare ASP.NET Web API 2 and ASP.NET Core](../architecture/porting-existing-aspnet-apps/webapi-differences.md) - Porting ASP.NET eBook REVIEW DRAFT
+
+## C# language
+
+### New articles
+
+- [Manipulate integral and floating point numbers in C\#](../csharp/tour-of-csharp/tutorials/numbers-in-csharp-local.md) - Finish get started tutorials
+
+### Updated articles
+
+- [Querying based on runtime state (C#)](../csharp/programming-guide/concepts/expression-trees/how-to-use-expression-trees-to-build-dynamic-queries.md) - Fix #6288
```

Comment:
`#6288` will be meaningless when published, as it's not the link. 
(I've noticed at least one more such "link" in this document below.)